sdk/cosmos/azure-cosmos/pyproject.toml

```diff
@@ -3,4 +3,4 @@ mypy = false
 pyright = false
 type_check_samples = false
 verifytypes = false
-pylint = false
```

Comment:
any chance we're ready to enable mypy as well?

Reply:
Not yet - but we're not far off now! That should come in "part 3" :)